tests/cad_import_and_curvature_refinement.cc

```diff
@@ -0,0 +1,28 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2014 by the deal.II authors
+//
+//    This file is subject to LGPL and may not be distributed
+//    without copyright and license information. Please refer
+//    to the file deal.II/doc/license.html for the  text  and
+//    further information on this license.
+//
+//-----------------------------------------------------------
+
+// Read a file in iges format, and write it out again in the same
+// format.
+
+#include "tests.h"
+#include "computational_domain.h"
+
+int main ()
+{
+  initlog(); 
+  ParameterAcceptor::initialize(SOURCE_DIR "/parameter_cad_import_and_curvature_refinement.prm","used.prm");
```

Comment:
Thanks, it was actually called before and after because I realized what you said and moved it after, but forgot to cancel the first. :D